.github/workflows/cd.yaml

```diff
@@ -0,0 +1,60 @@
+name: cd
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment:
+      name: xsnippet.org
+      url: https://xsnippet.org
+    concurrency: xsnippet.org
```

Comment:
Only one job at a time (even if it will be triggered concurrently).

Reply:
Cool trick!